data/lib/core/constants.lua

```diff
@@ -41,3 +41,19 @@ CYCLOPEDIA_SKILL_FIST = 11
 CYCLOPEDIA_SKILL_FISHING = 13
 
 CYCLOPEDIA_SKILL_AMOUNT = 8
+
+BLESS_TYPE_ADVENTURER = 1 -- not supported
+BLESS_TYPE_TWIST_OF_FATE = 2 -- not supported
+BLESS_TYPE_WISDOM_OF_SOLITUDE = 4
+BLESS_TYPE_SPARK_OF_PHOENIX = 8
+BLESS_TYPE_FIRE_OF_THE_SUNS = 16
+BLESS_TYPE_SPIRITUAL_SHIELD = 32
+BLESS_TYPE_THE_EMBRACE_OF_THE_WORLD = 64
+BLESS_TYPE_HEART_OF_MOUNTAIN = 128 -- not supported
+BLESS_TYPE_BLOOD_OF_MOUNTAIN = 256 -- not supported
```

Comment:
Why are these not supported? What do we need to support them? cc @omarcopires @ramon-bernardo let's add it to the backlog

Reply:
The bytes marked with --? are for AOL usage, and the other is for a red/black skull. Imagine also that a table would need to be created in the database for the history representing the bytes below!